buf.gen.yaml

```diff
@@ -2,9 +2,9 @@ version: v1
 managed:
   enabled: true
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/go:v1.28.1-1
+  - plugin: buf.build/protocolbuffers/go:v1.30.0
```

Comment:
1. Newer versions of the buf CLI use the `plugin` key for both remote/local plugin references
2. I updated to the latest versions, although can also omit the versions so it always uses the latest:

```yaml
plugins:
  - plugin: buf.build/protocolbuffers/go
```